packages/app/src/cli/services/context.ts

```diff
@@ -902,6 +923,33 @@ export function renderCurrentlyUsedConfigInfo({
   })
 }
 
+function renderAppLogsConfigInfo({
+  org,
+  appName,
+  devStore,
+  selectedStores,
+  configFile,
+  resetMessage,
+}: CurrentlyUsedConfigInfoOptions): void {
+  const items = [`App:             ${appName}`]
+  if (org) items.unshift(`Org:             ${org}`)
+  if (selectedStores && selectedStores.length > 0) {
+    selectedStores.forEach((storeUrl) => items.push(`Dev store:       ${storeUrl}`))
+  } else {
+    items.push(`Dev store:       ${devStore}`)
+  }
+
+  let body: TokenItem = [{list: {items}}]
+  if (resetMessage) body = [...body, '\n', ...resetMessage]
+
+  const fileName = configFile && getAppConfigurationFileName(configFile)
+
+  renderInfo({
+    headline: `Waiting for app logs...\n\n${configFile ? `Using ${fileName}:` : 'Using these settings:'}`,
+    body,
```

Comment:
A lot of this is very similar to today's render function, `renderCurrentlyUsedConfigInfo`. Is there an opportunity to reduce some of the code duplication between the two?

Reply:
Okay, I extracted some of the common code between the 2 methods out into a separate method getting called by both the render functions.